napari_allencell_annotator/_tests/widgets/template_list_test.py

```diff
@@ -1,13 +1,124 @@
 from unittest import mock
 from unittest.mock import MagicMock, patch, create_autospec
 
+from PyQt5.QtWidgets import QPushButton, QLineEdit
+
+from napari_allencell_annotator.model.combo_key import ComboKey
+
+from napari_allencell_annotator.model.key import Key
+from napari_allencell_annotator.widgets.template_item import ItemType
+
 from napari_allencell_annotator.widgets.template_list import (
     TemplateList,
     TemplateItem,
     QSpinBox,
     QCheckBox,
     QComboBox,
 )
+import pytest
+
+
+@pytest.fixture()
+def template_list(qtbot) -> TemplateList:
+    return TemplateList()
+
+
+def test_add_item_string(template_list: TemplateList) -> None:
+
+    # ACT
+    template_list.add_item("test_string", Key("string", "test_value"))
+
+    # ASSERT
+    assert template_list.count() == 1
+
+    item: TemplateItem = template_list.item(0)
+    assert item.name.text() == "test_string"
+    assert item.type == ItemType.STRING
+    assert item.default == "test_value"
+    assert isinstance(item.editable_widget, QLineEdit)
+    assert item.editable_widget.text() == "test_value"
```

Comment:
not needed, checked above

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/92/commits/7f445d5488a221349dad061156ede36c7ed49a37

Reply:
took out too many assertions. please keep all logic that is in the `add_item` function tested.

Reply:
added assertions back to tests in https://github.com/aics-int/napari-allencell-annotator/pull/92/commits/6eada85860f445d23ef9230cab3a6fd349e03505